ml-proto/src/spec/arithmetic.ml

```diff
@@ -42,6 +43,9 @@ sig
   val of_big_int_u : Big_int.big_int -> t
   val to_value : t -> value
   val of_value : int -> value -> t
+  val compare : t -> t -> int
```

Comment:
Ok. But question though, does val compare : 'a -> 'a -> int vs. val compare: t -> t -> int make a difference to specialization either way?